app/javascript/components/Results/ResultsCount/ResultsCount.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable camelcase */
```

Comment:
Why is this `eslint-disable` needed?

Reply:
It's needed to used the translation in here https://github.com/GSA/search-gov/pull/1368/files/da43b3e7dbb1a57314a5fdd9e0318665e52ddd6c#diff-14cde637abda389b07de93f02d2b612bc8a89c24b753322cc60b825693a01646R21